### PR TITLE
we are all stable:tm:

### PR DESCRIPTION
**Pull request by hYdos**

### CODE REVIEW

**Review by BoogieMonster1O1 — CHANGES_REQUESTED** [review]
this should only be applied in dev

**Comment by hYdos** [comment]
players dont like dealing with that screen either. it takes up loading time and is a pain

**Comment by hYdos** [comment]
also yes now have working fluids
image

File: src/main/java/org/dimdev/dimdoors/mixin/EntityMixin.java [hunk 1]
```diff
@@ -0,0 +1,51 @@
...
+import net.minecraft.world.World;
+import org.spongepowered.asm.mixin.Mixin;
+import org.spongepowered.asm.mixin.Shadow;
+import org.spongepowered.asm.mixin.injection.At;
+import org.spongepowered.asm.mixin.injection.Inject;
+import org.spongepowered.asm.mixin.injection.Redirect;
+import org.spongepowered.asm.mixin.injection.callback.CallbackInfoReturnable;
+
+@Mixin(value = Entity.class, priority = 1) // Stop Better End thinking. f*** your mod, only ours will load fluids properly
+public abstract class EntityMixin {
```
Comment (BoogieMonster1O1) [inline src/main/java/org/dimdev/dimdoors/mixin/EntityMixin.java:22]: no

**Comment by BoogieMonster1O1** [comment]
the warning is important, this is 1.17 and you can never be too careful

**Comment by BoogieMonster1O1** [comment]
plus there's mods that yeet it
nothing truly invasive goes in dimdoors

**Comment by Waterpicker** [comment]
Since this was a general bad idea. Closing

**Pull request closed without merging**